collective/tablepage/fields/image.py

```diff
@@ -32,6 +32,14 @@ class ImageField(FileField):
     edit_template = ViewPageTemplateFile('templates/image.pt')
     view_template = ViewPageTemplateFile('templates/image_view.pt')
 
+    def _get_obj_info(self, uuid):
```

Comment:
It would be good to have an explanatory docstring telling what this is doing, especially given that it seems to change existing behavior.